src/pages/CovidStatDetails.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Badge, Card } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import { useNavigate, useParams } from 'react-router';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowAltCircleLeft } from '@fortawesome/free-regular-svg-icons';
+
+const CovidStatDetails = () => {
+  const { id } = useParams();
+  const statsArray = useSelector((state) => state.covid.covidStats);
+  const detailArr = statsArray.find((stat) => parseInt(stat.id, 10) === parseInt(id, 10));
+  const navigate = useNavigate();
+  return (
+    <div className="mt-5">
+      <div className="d-flex flex-row justify-content-center gap-2">
+        <div className="text-left">
+          <FontAwesomeIcon icon={faArrowAltCircleLeft} size="2x" className="text-primary mb-2" onClick={() => navigate(-1)} style={{ cursor: 'pointer' }} />
```

Comment:
## Before
![image](https://user-images.githubusercontent.com/93039359/227443272-33537afa-f70e-46b2-a23f-61350f01c834.png)

## After
![image](https://user-images.githubusercontent.com/93039359/227442984-7673e7fe-f62c-48e2-905b-ea175012f04c.png)

- [ ] Great work on implementing some of the requirements but I guess you forgot about this change which was requested by the previous reviewer which was to make your back icon at the top left corner 🤔 Kindly make your icon at the top left corner as I did to make your design in the center and look professional 👍

Reply:
okay